code/datums/status_effects/buffs.dm

```diff
@@ -597,3 +597,22 @@
 
 /datum/status_effect/jump_jet/on_remove()
 	owner.RemoveElement(/datum/element/forced_gravity, 0)
+
+/// Makes the mob immune to radiation for a short bit to help with safely spawning in hazardous areas
+/datum/status_effect/radiation_immunity
+	id = "radiation_immunity"
+	duration = 1 MINUTES
+
+	alert_type = /atom/movable/screen/alert/status_effect/radiation_immunity
+
+/datum/status_effect/radiation_immunity/on_apply()
+	ADD_TRAIT(owner, TRAIT_RADIMMUNE, type)
+
```

Comment:
I can tell this PR is untested because this is not correct:
```suggestion
/datum/status_effect/radiation_immunity/on_apply()
	ADD_TRAIT(owner, TRAIT_RADIMMUNE, type)
	return TRUE
```

Reply:
Damn, caught red-handed. Not for a lack of trying though, but I can’t seem to force dynamic rulesets to trigger